dev/test/bundle.ts

```diff
@@ -24,6 +24,16 @@ import {
   verifyInstance,
 } from './util/helpers';
 import IBundleElement = firestore.IBundleElement;
+import IBundleMetadata = firestore.IBundleMetadata;
+
+export function verifyMetadata(
+  meta: IBundleMetadata,
+  expected: IBundleMetadata
```

Comment:
I was hoping you would pass in the expected state one at a time:

```
export function verifyMetadata(
  meta: IBundleMetadata,
  id: string,
  createTime: ReadTime,
  version: number,
  totalDocuments: 0
)
```

From that, you can simplify further. Extract a constant for the bundle ID, and use it everywhere. The version is also always a constant. So in the end you have:

```
export function verifyMetadata(
  meta: IBundleMetadata,
  createTime: ReadTime,
  totalDocuments: 0
)
```

You can then also compare each field directly:


```
  expect(meta.totalBytes).greaterThan(0);
  expect(meta.id).to.equal(BUNDLE_ID);
  expect(meta.version).to.equal(1);
  expect(meta.createTime).to.deep.equal(readTime.toProto());
  expect(meta.totalDocuments).to.equal(totalDocuments);
```

This removes the need to delete the field and ruins nasty surprises in the future.

Reply:
Done.